pom.xml

```diff
@@ -66,6 +66,29 @@
       </snapshots>
     </repository>
   </repositories>
+  <profiles>
+  <profile>
+      <id>fast</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <properties>
+        <test.excludeTags>slow,expensive</test.excludeTags>
+      </properties>
+    </profile>
+    <profile>
+      <id>slow</id>
+      <properties>
+        <test.excludeTags>expensive</test.excludeTags>
+      </properties>
+    </profile>
+    <profile>
+      <id>expensive</id>
+      <properties>
+        <test.excludeTags></test.excludeTags>
```

Comment:
Do you need this if it's empty?

Reply:
Maybe not, but I find it more expressive this way.